jkube-kit/common/src/main/java/org/eclipse/jkube/kit/common/util/ClassUtil.java

```diff
@@ -219,5 +225,32 @@ public static URLClassLoader createProjectClassLoader(List<String> elements, Kit
         // if somehow we incorrectly call this method
         return new URLClassLoader(new URL[]{});
     }
+
+    public static String[] getJavaClassPath() {
+        return System.getProperty(JAVA_CLASS_PATH).split(File.pathSeparator);
+    }
+
+    public static URL getFileFromJar(String jarPath, String filePath) throws IOException {
+        File jarFile = new File(jarPath);
+        if (jarFile.isFile()) {
+            try (final JarFile jar = new JarFile(jarFile)) {
+                final Enumeration<JarEntry> entries = jar.entries(); //gives ALL entries in jar
+                while(entries.hasMoreElements()) {
+                    final String name = entries.nextElement().getName();
+                    if (name.contains(filePath)) { //filter according to the path
+                        return new URL("jar:file:" + jarPath + "!" + appendPathSeparatorIfNotPresent(name));
+                    }
+                }
+            }
+        }
+        return null;
+    }
+
+    private static String appendPathSeparatorIfNotPresent(String fileName) {
+        if (!fileName.startsWith(File.separator)) {
+            return File.separator + fileName;
+        }
+        return fileName;
+    }
```

Comment:
Can be removed if https://github.com/eclipse/jkube/pull/528/files#r547953033 is applied